### ISSUE TITLE
Wildcard in MIME-types are not parsed

### ISSUE THREAD

**Issue by eleonoraortega** [mulesoft-labs/rajapa#26]
In the following example it is not parsed the wildcard 

``` yaml

#%RAML 1.0
title: wildcard
baseUri: http://wildcard.com/api/console/
resourceTypes:
  - organizationsCurrentOwnersUsername:
      usage: wildcard is not correctly parsed
/organizations/current: 
  get: 
    responses: 
      200: 
        description: OK
        body: 
          application/*:  
      400: 
        description: Mandatory query string parameter not being provided
      415: 
        description: The content type of the body is not supported
      404: 
        description: The organization cannot be found
      500: 
        description: Database current unavailable

```

Error:

```
- Unexpected key 'application/*' [line=13, col=11]
```

**Comment by eleonoraortega** [comment]
NOTE: Yaml parser is correctly parsing this example

**Comment by svacas** [comment]
fixed by 231733c679f373f958a96affb0ea60eacf75edc2
